salt/auth/django.py

```diff
@@ -28,9 +28,9 @@
 .. code-block:: python
 
     class SaltExternalAuthModel(models.Model):
-        user_fk = models.ForeignKey(auth.User)
-        minion_matcher = models.CharField()
-        minion_fn = models.CharField()
+        user_fk = models.ForeignKey(User, on_delete=models.CASCADE)
+        minion_or_fn_matcher = models.CharField(max_length=255)
+        minion_fn = models.CharField(max_length=255)
```

Comment:
If there's an assumption of Postgres as the database, [there's no reason to add max_length](https://www.depesz.com/2010/03/02/charx-vs-varcharx-vs-varchar-vs-text/).

Reply:
Ah... I see it's [required in Django's CharField](https://docs.djangoproject.com/en/2.2/ref/models/fields/#django.db.models.CharField)

Reply:
Apparently one needs to [make their own `CharField` to avoid Django's limit](https://stackoverflow.com/a/20116327)